### PR TITLE
Go 1.17 & Bullseye

### PR DESCRIPTION
**Pull request by jaredledvina**
Upgrading to Go 1.17, bullseye, and a `go get -u` for housekeeping 🧹

### CODE REVIEW

**Comment by jaredledvina** [comment]
I've completely forgotten how we build and deploy this....

File: go.mod [hunk 1]
```diff
@@ -10,9 +10,15 @@ require (
...
+	golang.org/x/oauth2 v0.0.0-20210819190943-2bc19b11175f // indirect
+	golang.org/x/sys v0.0.0-20210510120138-977fb7262007 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
-	google.golang.org/genproto v0.0.0-20210506142907-4a47615972c2 // indirect
-	google.golang.org/grpc v1.37.0 // indirect
+	google.golang.org/genproto v0.0.0-20210903162649-d08c68adba83 // indirect
+	google.golang.org/grpc v1.40.0 // indirect
+)
+
+require (
```
Comment (belak) [inline go.mod:20]: I like how they're moving indirect deps to a separate block, but I wish they migrated all of them

Comment (jaredledvina) [reply go.mod:20]: Hmm good point, I can just do that manually, interesting that they didn't do that, is there a reason you're aware of?

**Comment by belak** [comment]
> I've completely forgotten how we build and deploy this....

It's manual now that we don't get free builds on Docker Hub. Looking into github actions might be good as well.

```
# In this repo
$ docker build -t seabirdchat/seabird-stock-plugin .
$ docker push seabirdchat/seabird-stock-plugin

# In the infra repo
$ ansible-playbook site.yml -e update_docker_images=yes -t seabird-stock-plugin
```

File: go.mod (hunk 1 above)
Comment (belak) [reply go.mod:20]: Not that I'm aware of

**Pull request merged**